docs/S01-fundamentals/M1-cryptography/L2-pub-key-crypto-additional/index.md

```diff
@@ -32,7 +32,7 @@
 * [Video: End to End Encryption — Computerphile](https://www.youtube.com/watch?v=jkV1KEJGKRA){target=_blank} Introducing the general concepts behind using encryption in public networks
 * [Video: Gambling with Secrets — RSA Encryption](https://www.youtube.com/watch?v=vgTtHV04xRI){target=_blank}
 * [Article: What is Asymmetric Encryption?](https://dzone.com/articles/what-is-asymmetric-encryption-understand-with-simp-1){target=_blank}
-* [Article: Keeping Secrets Secret (BBC)](https://www.rigb.org/christmaslectures08/html/activities/keeping-secrets-secret.pdf#page=5){target=_blank} This is a valuable resource explaining, in simple visual terms, the modular arithmetic underpinning the security of public key cryptography, hashing (which we'll learn about next) and any other one-way or "trapdoor" functions. It illustrates how you cannot break a private key's encryption with brute-force but can easily validate it if you have the accompanying public key.
+* [Article: Keeping Secrets Secret (BBC)](https://web.archive.org/web/20200924112725/https://www.rigb.org/christmaslectures08/html/activities/keeping-secrets-secret.pdf){target=_blank} This is a valuable resource explaining, in simple visual terms, the modular arithmetic underpinning the security of public key cryptography, hashing (which we'll learn about next) and any other one-way or "trapdoor" functions. It illustrates how you cannot break a private key's encryption with brute-force but can easily validate it if you have the accompanying public key.
```

Comment:
Great catch on this one!

Reply:
That article was good, I needed to rescue it.